dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/FileDownloadHelper.java

```diff
@@ -349,7 +349,23 @@ public void clearRequestAccessFiles() {
 
     public void addMultipleFilesForRequestAccess(DataFile dataFile) {
         this.filesForRequestAccess.add(dataFile);
+    }
 
+    public Map<RestrictedTermsOfUseDTO, List<DataFile>> getFilesForRequestAccessByTermsOfUse() {
```

Comment:
I think Javadoc could be useful in this case, explaining why would you need this method (context), and not implementation details since it's easy to read.

Reply:
:+1: